apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
:( "The link you're trying to access can't be used to share files. Please ask the file owner to provide you with a shared link instead."